plugins/hydra_submitit_launcher/example/config.yaml

```diff
@@ -1,4 +1,11 @@
 defaults:
   - hydra/launcher: submitit
+hydra:
+  launcher:
+    params:
+      queue: auto
+      queue_parameters:
+        auto:
+          cluster: local
 
```

Comment:
This seems like a very verbose example.
What is the objective here? Is the current example not running locally by default?

I would prefer to keep the basic example using local explicitly.
We also need actual slurm testing here and a dedicated slurm example, but this is beyond the scope of this diff.

Reply:
It was using the local executor, not through the auto executor though. Ends up being the same but I feel that having the whole hierarchy explicitely written helps a lot when starting from nothing, it probably took me 15min to get there

Reply:
Specific plugin configuration discoverability is not great right now.
You can get the Hydra config (which contains them using `--cfg hydra`.